lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -547,9 +547,9 @@ void WebsocketTlsTPM::client_loop() {
     }
 
     // Print data for debug
-    EVLOG_info << "LWS connect with info "
-               << "port: [" << i.port << "] address: [" << i.address << "] path: [" << i.path << "] protocol: ["
-               << i.protocol << "]";
+    EVLOG_debug << "LWS connect with info "
```

Comment:
I would keep this as info, it only happens on a connect should not be too spammy.

Reply:
It also seems to happen on every reconnect attempt in case the connection does not work. But that is not the reason I removed this one. We also already print the same information in `Connecting to uri: etc` so I didn't really see a reason to print the same information twice.

Could you elaborate on what you are using from this?

Reply:
Sometimes there are some info missing from the uri (in certain contexts), maybe extend this one a bit with TPM and remove the second one?

Reply:
I do think the second one is more readable to leave in in the future. I will set this back to info for now, there is already a comment above it that it is for debug anyway so will hopefully be removed sometime in the future.